client/my-sites/importer/author-mapping-item.jsx

```diff
@@ -32,6 +32,15 @@ export default React.createClass( {
 		} ).isRequired
 	},
 
+	componentWillMount() {
+		const { hasSingleAuthor, onSelect } = this.props;
+
+		if ( hasSingleAuthor ) {
+			// Pre-selects the current user for author mapping, if the site only has one user.
```

Comment:
It seems pretty straightforward without the comment even. We might be just as well leaving it out